web/src/components/health/health-color-square.css

```diff
@@ -0,0 +1,23 @@
+.cell {
+  border-radius: 5px;
+  width: 1rem;
+  height: 1rem;
+  display: inline-block;
+  vertical-align: middle;
+  margin-right: 5px;
+}
+
+.gradient {
+  border-radius: 2px;
+  width: 5rem;
+  height: 5px;
+  display: inline-block;
+  margin-left: 5px;
+}
+
+.gradient .vertical-mark {
+  position: relative;
+  display: block;
+  height: 100%;
+  border-right: solid 1px black;
+}
```

Comment:
If possible, try to avoid px sizes here. There are some in the existing CSS and it can mess up depending the resolution you are using.

em / rem are better !

Reply:
done